Octokit/Models/Response/CommitCommentReaction.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Octokit.Internal;
+using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    public enum Reaction
+    {
+        [Parameter(Value = "plus_one")]
+        Plus1 = 0,
+        [Parameter(Value = "minus_one")]
+        Minus1 = 1,
+        [Parameter(Value = "laugh")]
+        Laugh = 2,
```

Comment:
If you look at other enums in octokit.net we dont normally assign integer values to the enum members